### PR TITLE
Funkychem, so realll :3

### PR DESCRIPTION
**Pull request by TooSillyFennec**
## About the PR
Many new chems replace old chems, new toxins and a new tree of top level chems.

## Why / Balance
All of new medchems are based off 13 and are there to add more variation and fun with experimenting. Max. Metab increased to 3 to allow more skill expression in making mixes.

Key Changes:

Metabolism rate on med chems 0.5 > 0.2 - Meds should take longer than 30s to do their effects. This also incentivises making faster mixes.

Fuel in Chem Dispenser - Great for recipes as it opens up many chems to make complex recipes without needing the 1 minute trip to maints. [And 13 chems use the chems it makes veryyy often!]

Jugs - Nuked from the chemvend, but they have 400u max to compensate chem. Returns the game to pre jug meta. Doctors may actually request their own meds, giving chem extra things to do when the general meds are done. 

Medkits - All of them have much more stuff, and better stuff. Most medkits were laughably bad and maybe treated 2-3 patients after the fact the meds inside. Now, they are stronger, last muchhh longer and better filled while still being weaker than a good chem team by far, using the weaker meds unmixed.

Rest are additions / flavour changes.

## Technical details
iconic "fuck this i'd rather not take a year to create this" strategy!!! Mapfixes.yml generally handles me needing to fuck around and remap every fucking map the game has. same exact items, just the ids used pre-PR- less effort, 0 effect on players.

## Requirements

- [Guh] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [Guh] I have added media to this PR or it does not require an ingame showcase.

## "Roadmap"
Not really, but things that are needed for a completed chemistry. These are:
- inverse chems [Centrifuge/Electrolysis maybe can be used? i may do this myself.]
- Chems that work on a property, like their heat. [AKA Seiver really]
- Chems with item/structure interactions [AKA Thermite]
- More advanced drugs, and addictions
- Reworked Pyrotechnics [Kaboommm!!!]
- Chem Factories
- Surgery Chems [AKA, Surgery Speed]
- organ damage chems / effects
- Closer ties to Science [Chem should need them for more advanced things, and Sci should have experiments based on chems / pyrotechnics]
- Energy Chem 

**Changelog**

:cl:
- add: Many top level chems created by mixing a new chemical, Liquid Exotium.
- add: Many new toxins have been added. Test out Heparin [Bleed], Rhigoxane [Cold], Spewium, and Pestiline [VERY slow cellular toxin]
- add: Bicaridine, Kelotane, and Dermaline have been replaced by 6 new chemicals, as well as five other new chemicals, some replacing older medical chems. Treat /Patients/ with /SS13 Rip off/ Chemicals today! - Central Command Automated Advertiser
- tweak: Normal species can metabolise 3 chemicals now, have fun making mixes!
- tweak: Medical Chemicals now metabolise at the rate of 0.2, making them last longer, and have higher overdoses. Basic Med Chems od at 20, and Advanced Med Chems at 15. Medicine heals slower to compensate.
- tweak: Medical patches have been re-balanced completely. And they also don't have spelling errors... yay.
- tweak: Chem Dispensers now have welding fuel.
- tweak: Medkits have been changed heavily, to be stronger and have much more contents. Have fun healing more than 1 person now!

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/Medical/AdvancedBrute.xml [hunk 1]
```diff
@@ -10,19 +10,19 @@
 
   # Razorium
 
-  [color=red]USE CAUTION[/color] when using the aforementioned specialized brute chemicals. Should one mix together with a different type of brute chemical, the resulting mixture will create a dangerous chemical known as Razorium!
+  [color=red]USE CAUTION[/color] when using the aforementioned specialized brute chemicals. Should one mix together with a different type of advanced brute chemical, the resulting mixture will create a dangerous chemical known as Razorium!
   <GuideReagentEmbed Reagent="Razorium"/>
-  To avoid this, make sure the patient is done metabolizing one brute medication before giving them another. Keep an eye on the rate their health is changing.
+  To avoid this, make sure the patient is done metabolizing one brut advanced e medication before giving them another. Keep an eye on the rate their health is changing.
```
Comment (Rainbeon) [inline Resources/ServerInfo/Guidebook/Medical/AdvancedBrute.xml:15]: I don't think you meant to put advanced in this particular location

File: Resources/Prototypes/Catalog/Fills/Items/firstaidkits.yml
Comment (DevilishMilk) [inline Resources/Prototypes/Catalog/Fills/Items/firstaidkits.yml:1]: Basic first aid kit should still contain gauze to stop people from bleeding out

**Comment by Rainbeon** [comment]
I have very large concerns about the changes to "Jug Chem" (especially in the context of chem economy. Removing ALL of the carbon/oxygen/etc from the vend for a measly 200u more may cause severe bottle-necking like was seen in the early days of LimChem). As is there are already problems with resource scarcity and these problems *very often* discourage mixing due to Chemists wanting to maximize resource efficiency so they can play the game, which seems like the opposite of what you want. We've already had thoughts of 13-style Energy-Chem, anyway.

"Exotic Reagent" is a little bit too... bland and un-chemically of a name? It feels very out of place, especially if it's going to be something that is decently important.

The rest I'm kind of skeptical of how fun it'll be for Chem/Docs/Patients alike but would need to see how they work in practice before giving any final thoughts.

One thing I do like is the 3 chemical limit, 2 is just too restrictive and kills any form of pre-mixing you could possibly do, so this is a good change.

**Comment by taydeo** [comment]
I agree with Rainbeon on the oxygen issue, restricting chemists in this way only serves to unnecessarily complicate their job. 

I also want to see Exotic Reagent given a proper name.

ALSO, test fail IS related to the PR, if someone can look into that, that'd be great.

**Comment by corresp0nd** [comment]
omg 3 chem metab yayyy!!! 
i would probably have to see most of this in-game to see how i truly feel about it, esp bc i am not a ss13 gamer and never touched that medical system. i generally like the idea of changing gears to mixes rather than jug bags, since it offers some extra complexity to chem & gives some uniqueness to how each chemist actually does their job. 

i do really agree that removing all of the jugs from the chemvend sucksss though. i think it's just going to result in chemistry begging cargo for the chemical crates shift start, and then... they still have jugs. or they don't have jugs and no chems. maybe this isn't as big of a problem since the way brutes are made is changed, and those are really what eats up carbon? again, i probably need to playtest this to be sure. also, the price of chemvend restocks should be suuper lowered. maybe you did change it and im blind though
as a chemist i would probably shift gears towards making patches or pills (esp since there was an upstream change to make pills resistant to explosions), but that has consistently been Really Annoying for chemistry to do because you need a clean chem master buffer. if we really want to get away from the jug meta, i think we should consider some sort of extra machine to specifically fill pills/patches. or change how it's done in the chem masters currently, but reusing the code in another machine would probably just be easier to do.

anywayz i think it would overall be best if we playtest it for a day or two and see how it goes. i like it, i just need to see how it goes before i give any concrete thoughts

File: Resources/Locale/en-US/_Funkystation/chemexpansion.ftl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+reagent-desc-penteticacid = A complicated anti-toxin solution that quickly flushes out toxins and heals the damage left behind. Overdose causes extreme pain.
+
+reagent-name-granibitulari = Granibitulari
+reagent-desc-granibitulari = A Commonly sold medicine for all types of injuries, but extremely slow acting.
+
+reagent-name-heparin = Heparin
+reagent-desc-heparin = A Fast-Acting toxin that quickly opens wounds on a body, making them bleed rapidly. 
+
+reagent-name-rhigoxane = Rhigoxane
+reagent-desc-rhigoxane = That looks so cold and refreshing... don't take a sip, it's pretty warm... and you shall be cold as you thought it was if ingested.
```
Comment (sheepism1) [inline Resources/Locale/en-US/_Funkystation/chemexpansion.ftl:38]: better documentation of effects please.

File: Resources/Locale/en-US/_Funkystation/chemexpansion.ftl [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+reagent-desc-pestiline = This exotic and strange toxin is rumored to be the essence of sickness itself.
+
+reagent-name-exoticreagent = Exotic Reagent
+reagent-desc-exoticreagent = You've never even heard of it before, unless you are a chemist. Then, it's secrets are in your hands, and it's time to use the true power of chemistry.
+
+reagent-name-stasicillin = Stasicillin
+reagent-desc-stasicillin = A very rare chemical used by both top-level medbays and nanotrasen secret agents for both the effective injury treatment and it's side effect, sending the consumer to sleep.
+
+reagent-name-negacordrazine = Negacordrazine
+reagent-desc-negacordrazine = A strange, completely reversed tricordrazine. Heals nearly all damage types above 30 Damage.
```
Comment (sheepism1) [inline Resources/Locale/en-US/_Funkystation/chemexpansion.ftl:53]: making it poisonous only above certain damage would be Fun.

File: Resources/Locale/en-US/_Funkystation/chemexpansion.ftl [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+reagent-desc-rhigoxane = That looks so cold and refreshing... don't take a sip, it's pretty warm... and you shall be cold as you thought it was if ingested.
+
+reagent-name-spewium = Spewium
+reagent-desc-spewium = Even smelling it warrants a vomit comet.
+
+reagent-name-pestiline = Pestiline
+reagent-desc-pestiline = This exotic and strange toxin is rumored to be the essence of sickness itself.
+
+reagent-name-exoticreagent = Exotic Reagent
+reagent-desc-exoticreagent = You've never even heard of it before, unless you are a chemist. Then, it's secrets are in your hands, and it's time to use the true power of chemistry.
```
Comment (sheepism1) [inline Resources/Locale/en-US/_Funkystation/chemexpansion.ftl:47]: how about "A precursor to many different chemicals, allowing you to unlock the next level of medical science."

File: Resources/Prototypes/Hydroponics/randomChemicals.yml [hunk 5]
```diff
@@ -10,16 +10,19 @@
...
     reagents:
     - Cognizine
     - Toxin
     - Leporazine
     - Epinephrine
+    - Convermol # Funkychem
     - CarpoToxin
     - SulfuricAcid
     - Licoxide
+    - Pestiline # Funkychem
```
Comment (sheepism1) [inline Resources/Prototypes/Hydroponics/randomChemicals.yml:25]: this should REALLY be in toxins.yml

**Review by sheepism1 — CHANGES_REQUESTED** [review]
Needs balance work

**Comment by TooSillyFennec** [comment]
Fixed a few things:

Jugs at 500u - At an average chem department of 2 Chem Dispensers / 1 Chemvend or 3 / 2, That's 3 or 4.5 jugs of chemicals comparatively to 3 / 6. Also, recipes are less... heavy on certain chems. Barely changes actual chem amount, some chemists actually gain now, at least the ones at the 3:1 Ratio on the map, and there's a few. But yeah, 400 was kinda low.

Medkit Gauze - Yeah makes sense, not everyone's gonna have a cautery or emergency pen to handle a bleeding patient.

Exotic Renamed - The original name was on the nose in the "Strange Reagent" style, this one's sci-fi bullshittery. Fits the somewhat jokey name intended and has more flavour, generally a good idea.

oh and chemvend restocks have been cheapened to 2000- still encourages "Make it yourself!" without being stupidly expensive.

**Comment by Rainbeon** [comment]
Doesn't ramping up jug size just lead to super chem storage? Like just taking them out and hauling around 500u of Chems in a 2x2 inventory space.

I'm not really sure what the Jug size/vend changes in particular are trying to accomplish. From what I can see they're now going to give ~the same amount of Chems, and the only difference is you can carry around a ludicrous amount on your person.

If you want to change how Chem distributes things, changing Jug stuff doesn't really help that much because the reason "Jug Meta" exists is because it's the most convenient due to lacking features in distributing the others. (SmartFridge, ChemMaster pill/bottle-making rework). I'm not really sure what else it could be for.

**Comment by TooSillyFennec** [comment]
> Doesn't ramping up jug size just lead to super chem storage? Like just taking them out and hauling around 500u of Chems in a 2x2 inventory space.
> 
> I'm not really sure what the Jug size/vend changes in particular are trying to accomplish. From what I can see they're now going to give ~the same amount of Chems, and the only difference is you can carry around a ludicrous amount on your person.
> 
> If you want to change how Chem distributes things, changing Jug stuff doesn't really help that much because the reason "Jug Meta" exists is because it's the most convenient due to lacking features in distributing the others. (functioning SmartFridge, ChemMaster pill/bottle-making rework). I'm not really sure what else it could be for.
> 
> I think the Test Merge would likely stick better without the Jug changes, since everything else looks cool and fun, just this specific part I think is going to be an issue.

Drawing was removed and much less spare jugs so chem can't just make 20 jugs and hand them to the docs, but honestly? yeah let's just test 0 drawing, 0 transfering and leave the jugs the same. if need be, a better solution can be made later if people still do the juggening:tm:

**Comment by Rainbeon** [comment]
>Drawing was removed and much less spare jugs so chem can't just make 20 jugs and hand them to the docs, but honestly? yeah let's just test 0 drawing, 0 transfering and leave the jugs the same. if need be, a better solution can be made later if people still do the juggening:tm:

Well, what I am saying then is just removing stuff doesn't fix the problem. We can do this but we shouldn't deteriorate the gameplay experience without implementing the alternative (upstream syndrome), the machines responsible for making pills and bottles need to be atleast somewhat implemented alongside it. It doesn't seem like a good idea to include for this specific PR though.

**Comment by taydeo** [comment]
I'll be willing to pitch in for a solution to solve this, seeing that removing drawing from jugs is not a solution to the problem. I'd like for the jug changes to be reverted, so people can draw from them again.

**Comment by TooSillyFennec** [comment]
> I'll be willing to pitch in for a solution to solve this, seeing that removing drawing from jugs is not a solution to the problem. I'd like for the jug changes to be reverted, so people can draw from them again.

the reason was "medbay before jugs was better for doctors as they had some actual skill expression of what chems they want and which method, jugs are a problem" but being real, yeah it's fine to revert it as it's secondary to the PR's main focus, but I REALLY think medbay needs to move away from Jug > Click > Patient > Click as the main gameplay, and surgery hasn't been very heavily required yet to really change that.

**Comment by taydeo** [comment]
dw about this ill handle it when it comes time to revert it 💯

**Comment by taydeo** [comment]
Being undone pending rework, as we know what we need to change. Thanks for playtesting if you see this!

**Pull request merged**